docs/diagrams/ModelClassDiagram.puml

```diff
@@ -27,9 +27,15 @@ Class Address
 Class Email
 Class Name
 Class Phone
-Class Tag
 Class MedicalHistory
-Class MedicalHistoryEntry
+Class MedicalEntry
+Class EntryList
+class "{abstract}\nEntry" as Entry
+class Entry.Some<MedicalEntry> MODEL_COLOR {
+}
+
+class Entry.Empty<Object> MODEL_COLOR {
+}
```

Comment:
![image](https://user-images.githubusercontent.com/24633766/140641267-ca8b3040-f72f-4c5e-a56e-3d676aa525db.png)
This might be too detailed for an overview

Reply:
Reduced detail by putting a note